lg_mirror/test/test_mirror_scene.py

```diff
@@ -38,7 +38,7 @@
     {
       "activity": "browser",
       "activity_config": {
-        "preload": false
+        "preload": true
```

Comment:
I've been testing mirroring with preload set to `true` which relies on onDomReady or onDomContentLoaded (one of them) and it was not doing it's thing in the right moment.